src/components/DayPicker.jsx

```diff
@@ -68,11 +72,17 @@ const defaultProps = {
   monthFormat: 'MMMM YYYY',
 };
 
+const triggerInitialVisibleMonth = (props) => (
+  props.hidden ? moment() : props.initialVisibleMonth()
```

Comment:
Oh sorry about that! I think if it's only used once, we should not pull it out. :) Let's go with that.